docs/source/routing/self-hosted/index.mdx

```diff
@@ -21,7 +21,7 @@ flowchart LR;
 
 <Note>
 
-Self-hosting the GraphOS Router requires a current GraphOS [Enterprise or Free plan](https://www.apollographql.com/pricing).
+In order to use paid features of the Router, a GraphOS [plan](https://www.apollographql.com/pricing) is required. Learn more about [GraphOS Router features](/graphos/routing/graphos-features).
```

Comment:
AFAIK, not all GraphOS features require a _paid_ plan. (A free plan will suffice.)

```suggestion
Certain router features require a GraphOS [plan](https://www.apollographql.com/pricing). [Learn more.](/graphos/routing/graphos-features).
```